src/frontend/frontend/cache_models.py

```
@@ -43,22 +43,21 @@ def __init__(
-                total_count=self._total_count,
-            )
-        return result
+            return super().__getitem__(item)
+        return super().__getitem__(item)
 
     @property
-    def timeout(self) -> float | int:
+    def timeout(self) -> int:
         if self and hasattr(self[0], "_cache_timeout"):
-            return getattr(self[0], "_cache_timeout")
+            return int(getattr(self[0], "_cache_timeout"))
```

Ruff (0.15.20)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)